source/ranges.tex

```diff
@@ -1752,39 +1780,39 @@
 template<class I>
   concept @\placeholder{Advanceable}@ =
     @\placeholdernc{Decrementable}@<I> && StrictTotallyOrdered<I> &&
-    requires(I i, const I j, const iter_difference_t<I> n) {
+    requires(I i, const I j, const @\placeholdernc{IOTA_DIFF_T}@(I) n) {
       { i += n } -> Same<I&>;
       { i -= n } -> Same<I&>;
-      { j +  n } -> Same<I>;
-      { n +  j } -> Same<I>;
-      { j -  n } -> Same<I>;
-      { j -  j } -> Same<iter_difference_t<I>>;
+      I(j + n);
+      I(n + j);
+      I(j - n);
+      { j -  j } -> ConvertibleTo<@\placeholdernc{IOTA_DIFF_T}@(I)>;
```

Comment:
The extra space for alignment before the second operand could be removed, or added to the above expressions.

Reply:
Fixed.